packages/panels/src/Resources/Pages/CreateRecord.php

```diff
@@ -89,7 +92,11 @@ public function create(bool $another = false): void
             $this->form->model($this->getRecord())->saveRelationships();
 
             $this->callHook('afterCreate');
+
+            DB::commit();
         } catch (Halt $exception) {
+            DB::rollBack();
```

Comment:
Halt is used to stop the creation process, and can happen after the state is saved. I think for the pages, you need to just wrap the `getState()` until `saveRelationships()` / `handleRecordUpdate()`

Also, check my changes on your other PR, I think there are quite a few other files that require this treatment, whenever we call `->save()`

Reply:
@danharrin I made some changes in the PR and covered the locations where multiple database actions where involved in the saving process. For now I only skipped the Import from cover them with database transactions as this process maybe should have special attention in the future and can't be only protected with a database transaction.